README.md

```diff
@@ -12,64 +12,157 @@ A resource may classify its messages into groupings,
 and it may include additional data or metadata relating to them.
 It should be possible to represent a resource in a text-based human-friendly manner.
 
-### How to Contribute
-
-- **Issues** are for specific aspects of the resulting specification.
-  Discussions here should be focused, and stay on topic.
-  New issues must include a sufficient initial description to introduce themselves,
-  and may contain an exploration of possible solutions to them.
-- **Pull requests** help develop and define the eventual specification.
-  Accepting and merging a PR should not be taken as the final word on any topic,
-  but each change should be an improvement on the previous.
-  Each PR should only apply one change that may be squashed to the `main` branch.
-  Text content should use [semantic line breaks](https://sembr.org/).
-  A PR may close one or more issues, but this is not required.
-- **Discussions** are freeform, and may well refer to multiple issues or topics.
-  This is also the forum in which we should develop further our ways of working,
-  which may also include interactions and forums outside this repository,
-  such as occasional video calls.
-
-We welcome your participation and interest!
-
-## Syntax
-
-As currently specified, a message resource looks like this (syntax highlighting only approximate):
+This work was [presented](https://www.youtube.com/watch?v=ksgm_B-uUCU)
+at the [2024 Unicode Tech Workshop](https://www.unicode.org/events/utw/2024/).
+
+### Why?
+
+The prior work on a new message format has identified the following challenges
+that go beyond or arise from defining the syntax and behavior of a single message,
+but which are not well addressed by existing resource formats:
+
+- The MessageFormat 2 message syntax is naturally multi-line,
+  and multi-line values are not easy to use in many of the current resource formats.
+- Comments and metadata are important for translators,
+  but (mostly) not relevant for formatting.
```

Comment:
Maybe break these apart. Requirements are _interesting_.

```suggestion
- Comments are an important means for translators and developers
   to communicate with one-another
   or for maintenance, but are irrelevant to the retrieval and display of the message.
- Metadata likewise provides information that assists the translation process,
   especially where tooling or CAT technology is used.
```

Reply:
Expanding on this a bit makes sense, but I don't think these should be separated. Both of these (and metadata in particular) can be defined either manually or by tools, and should be handled the same way.

```suggestion
- Comments and metadata (set either by automation or manually)
  are an important means for translators and developers to communicated with one-another,
  but are irrelevant to the retrieval and display of the message.
```